2017-11-17 06:15:21,069 - streamparse.stdout - INFO - Could not create tcount
2017-11-17 06:15:21,181 - streamparse.stdout - INFO - could not create table
2017-11-17 06:15:21,181 - streamparse.stdout - INFO - ucount = 
2017-11-17 06:15:21,182 - streamparse.stdout - INFO - 1
2017-11-17 06:15:21,182 - streamparse.stdout - INFO - uword = 
2017-11-17 06:15:21,182 - streamparse.stdout - INFO - Loving
2017-11-17 06:15:21,197 - streamparse.storm.component - ERROR - Exception in WordCounter.run()
Traceback (most recent call last):
  File "/usr/lib/python2.7/site-packages/streamparse-2.1.4-py2.7.egg/streamparse/storm/component.py", line 489, in run
    self._run()
  File "/usr/lib/python2.7/site-packages/streamparse-2.1.4-py2.7.egg/streamparse/storm/bolt.py", line 219, in _run
    self.process(tup)
  File "bolts/wordcount.py", line 67, in process
    uCount, uWord) )
InternalError: current transaction is aborted, commands ignored until end of transaction block

